SubProject.cmake

```diff
@@ -31,6 +31,8 @@
 #   example for continuous integration builds
 # - INSTALL_PACKAGES: command line cache variable which will "apt-get" or
 #   "port install" the known system packages. Will be unset after installation.
+# - ROOT_SOURCE_DIR: When set, the source code of subprojects will be
+#   downloaded in this path instead of CMAKE_SOURCE_DIR.
```

Comment:
Please call it COMMON_SOURCE_DIR (consistency with other common stuff, plus use case is inherently clear)